.github/workflows/ci_check_license.yml

```diff
@@ -20,17 +20,17 @@ name: InLong Check License Header
 on: [ push, pull_request ]
 
 jobs:
-  build:
-    name: Check License Header
+  check-license:
+    name: Check license header
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
-      - name: Check License Header
-        uses: apache/skywalking-eyes@2f5c0bce5575a25916d6b809c4dac84e6c750d47
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Check license header
+        uses: apache/skywalking-eyes@v0.3.0
```

Comment:
@gosonzhang Can the version of [apache/skywalking-eyes](https://github.com/apache/skywalking-eyes) be changed? Here is the [v3 change log](https://github.com/apache/skywalking-eyes/releases/tag/v0.3.0).

Reply:
ok